.github/workflows/tag-build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build with tag
+
+on:
+  push:
+    tags: [ v*.*.* ]
+
+jobs:
+
+  jenkins-build:
+    name: Deploys new release to stage
+    runs-on: ubuntu-latest
+    steps:
+     - name: Tag name
+       id: extract_tag
+       run: | 
+         VERSION_TAG=${GITHUB_REF##*/}
+         echo ::set-output name=versionTag::${VERSION_TAG}
+     - name: Jenkins webhook
+       uses: enflo/curl-action@fabe347922c7a9e88bafa15c4b7d6326ea802695
+       with:
+         curl: 'https://${{ secrets.JENKINS_USER }}@${{ secrets.JENKINS_HOST }}/job/DeployUrsus/buildWithParameters?GIT_BRANCH=${{ steps.extract_tag.outputs.versionTag }}&cause=GitHub+Action+Build+with+tag&DEPLOY_HOST=s-w-sinaimanuscripts01.library.ucla.edu&token=${{ secrets.JENKINS_API_SINAI_TOKEN }}'
```

Comment:
Can use `secrets.JENKINS_AUTH_TOKEN` instead of `secrets.JENKINS_API_SINAI_TOKEN`